InvoiceMaker/Views/FeeLineItem/Create.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@model InvoiceMaker.FormModels.CreateFeeLineItem
+
+@{
+    ViewBag.Title = "Create";
+}
+
+<h2>Add a FeeLine Item</h2>
+
+@using (Html.BeginForm("Create", "FeeLineItem", FormMethod.Post, new { @class = "pure-form pure-form-aligned" }))
+{
+    @Html.AntiForgeryToken()
```

Comment:
Don't forget to add the `ValidateAntiForgeryToken` attribute to the POST action method.